examples/angular2/readme.md

```diff
@@ -0,0 +1,32 @@
+# Angular 2 • [TodoMVC](http://todomvc.com)
+
+> Angular is a development platform for creating applications using modern web standards. Angular includes a wealth of essential features such as mobile gestures, animations, filtering, routing, data binding, security, internationalization, and beautiful UI components. It's extremely modular, lightweight, and easy to learn.
```

Comment:
This is a general description of angular. I'd like the readme to clarify how Angular and Angular 2 are different, at least for the time this example won't replace the main one.